docs/reference/release-notes/v2.28/index.md

```diff
@@ -0,0 +1,98 @@
+---
+date: "2020-12-01"
+version: "2.28.0"
+---
+
+# [v2.28](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0) (December 2020 #1)
+
+---
+
+Welcome to `gatsby@2.28.0` release (December 2020 #1).
+
+Key highlights of this release:
+
+- [New `gatsby new`](#new-gatsby-new) - new, interactive way to create a Gatsby site
+- [Improved `react-refresh` integration](#improved-react-refresh-integration)
+- [Feature flags in gatsby-config](#feature-flags-in-gatsby-config) - set your feature toggles without environment variables
+- [Experimental: Lazy images in develop](#experimental-lazy-images-in-develop) - run image transformations only when they are needed by browser
+- [Removed experimental lazy page bundling](#removed-experimental-lazy-page-bundling)
+- [Notable bugfixes](#notable-bugfixes)
+
+Sneak peek to next releases:
+
+- [Less aggressive cache invalidation](#less-aggressive-cache-invalidation)
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](../v2.27/index.md)<br>
+[Full changelog](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0)
+
+## new `gatsby new`
+
+In previous release we added [interactive way of scaffolding new gatsby project](../v2.27/index.md#create-gatsby) (with `npm init gatsby`). Now we adjusted `gatsby new` command when no arguments are passed to also use the same flow.
+
+## Improved `react-refresh` integration
+
+Gatsby had some support for `react-refresh` (or Fast Refresh as it can be referred to) already. This release improves it and adds better error overlays to be able to fix runtime and compilation error faster!
+
+To use it:
+
+```shell
+GATSBY_HOT_LOADER=fast-refresh gatsby develop
+```
+
+## Feature flags in `gatsby-config`
+
+Gatsby traditionally used environment variables to use various modes or enable experimental features. This was getting the job done but was far from pleasant experience. Setting environment variables through command line doesn't have uniform syntax for all operating systems (it's different for Windows and different for Linux or MacOS), it often forced using npm scripts when user wanted to use it permanently. Using flags in `gatsby-config.js` will allow to set and forget and will allow for tracking it in source control so the mode is consistent for everyone working on shared source code.
+
+To use those, add `flags` to your `gatsby-config.js`:
+
+```js
+// In your gatsby-config.js
+module.exports = {
+  // your existing config
+  flags: {
+    FAST_DEV: true,
+  },
+}
+```
+
+Currently supported flags:
+
+- `DEV_SSR` - SSR pages on full reloads during develop. Helps you detect SSR bugs and fix them without needing to do full builds.
```

Comment:
```suggestion
- `DEV_SSR` - Server-side render (SSR) pages on full reloads during develop. Helps you detect SSR bugs and fix them without needing to do full builds.
```

Reply:
(Why: explain the acronym, even though this is targeted at developers, it's still useful to clarify!)

Reply:
Good point. This was straight copied from https://github.com/gatsbyjs/gatsby/blob/9c8f78842784c6882953d2f5c72dd71d61ba3e29/packages/gatsby/src/utils/flags.ts#L41 so it also might make sense to apply same change there. cc @KyleAMathews